### DISCUSSION TITLE
dragonfly minimum requirements

### DISCUSSION THREAD

**Discussion by araera111** [dragonflydb/dragonfly#3096] [Q&A]
I am thinking of installing dragonfly on a virtual server called proxmox.
Of course, I know that memory, CPU, and HDD requirements will vary depending on the service.
Still, for a very simple system, what would be the minimum requirements?

**Comment by romange** [comment]
I would say 1gb,  or at least 250mb per cpu thread. You can limit the number of threads by --proactor_threads argument

**Comment by araera111** [comment]
thank you!
